website/widgets/form/image_annotator.md

```diff
@@ -86,3 +86,49 @@ The legend can be brought up by clicking the Info icon in the top right toolbar.
 |          **r**           |                     Select rectangle tool                     |
 |          **p**           |                       Select polygon tool                     |
 |          **s**           |                     Activate selection tool                   |
+
+## Click event
+
+Sometimes you may want to handle the annotating yourself and this is where `click` event comes into place. By setting `events=['click']` prop, the event is fired every time you click on the image canvas while using a `rect` or `polygon` tool and contains the `x` and `y` coordinates of the clicked point, e.g. `{x: 100, y: 120}`. The event can be accessed through `q.events.<image_annotator_name>.click`, where `image_annotator_name` is the `name` attribute of the image annotator component.
```

Comment:
No need to reexplain `events` here. This section should serve as a visual guide to what is possible and how it looks. Since events are "invisible", you can say handling custom click / toolchange events is possible.

Also let's try to use passive voice when possible instead of "Sometimes you want..."